src/layout/Dashboard.tsx

```diff
@@ -98,7 +98,8 @@ const Dashboard = () => {
         isResizable={true}
         panelContent={drawerConsole}
         position="bottom"
-        style={{ maxHeight: window.innerHeight - 28 }}
+        // 29px is the size of bottom Console banner
+        style={{ height: 'calc(100vh - 29px)' }}
```

Comment:
for another time: might be nice to provide a variable/constant for the 29px to know from where it is coming from

Reply:
@apupier yeah, It'd be good to reactor it a little bit and get rid of the magic numbers.